src/goto-symex/goto_symex.cpp

```diff
@@ -503,3 +508,77 @@ bool goto_symext::constant_propagate_string_substring(
 
   return true;
 }
+
+bool goto_symext::constant_propagate_integer_to_string(
```

Comment:
Document the expected arguments at the top of this function -- looks like `string_length, string_data, value, [base]`?

Reply:
Done